static/js/components/widgets/RelationField.tsx

```diff
@@ -177,7 +177,7 @@ export default function RelationField(props: Props): JSX.Element {
       const url = siteApiContentListingUrl
         .query({
           detailed_list:   true,
-          published:       true,
+          published:       crossSite ? true : false,
```

Comment:
I think this may not be what we want to do here - the way this parameter is implemented is that you pass `published: boolean` and it returns only items which match that publication status - in other words, passing `published: false` will return only unpublished content. So if you want to not filter to only published content, the thing to do would be to omit the parameter.

So I think instead what you probably want to do is

```suggestion
          ...(crossSite ? { published: true } : {})
```